src/auth/project-config.ts

```diff
@@ -121,10 +162,14 @@ export class ProjectConfig {
     // JSON serialization
     const json = {
       smsRegionConfig: deepCopy(this.smsRegionConfig),
+      multiFactorConfig: deepCopy(this.multiFactorConfig),
```

Comment:
"deepCopy(this.multiFactorConfig_)" ?

Reply:
Since `multiFactorConfig` is the getter function, shouldn't we use it that way to fetch `multiFactorConfig_` values in that?